app/controllers/users/additional_mfa_required_controller.rb

```diff
@@ -3,6 +3,8 @@ class AdditionalMfaRequiredController < ApplicationController
     include SecureHeadersConcern
     extend ActiveSupport::Concern
 
+    before_action :confirm_user_fully_uathenticated
```

Comment:
```suggestion
    before_action :confirm_user_fully_authenticated
```

Does the method need to be defined?

```ruby
def confirm_user_fully_authenticated
  return confirm_two_factor_authenticated unless user_fully_authenticated?
end
```